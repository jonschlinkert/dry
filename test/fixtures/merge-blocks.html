{% extends "merge-blocks" %}

<!-- block -->

{% block "a" %}
a: child
  {% block "b" %}
  b: child inside
  {% endblock %}
{% endblock %}

<!-- block -->

{% block "c" %}
c: child
{% endblock %}

{% block "c.d" %}
c.d: child
{% endblock %}

<!-- block -->

{% block "e" %}
e: child
  {% block "f" %}
  f: child inside
  {% endblock %}
{% endblock %}

{% block "e.f" %}
e.f: child outside
{% endblock %}

<!-- block -->

{% block merge="g" %}
g: child
{% endblock %}

{% block "g.h" %}
g.h: child outside
{% endblock %}

<!-- block -->

{% block merge="i" %}
i: child
{% endblock %}