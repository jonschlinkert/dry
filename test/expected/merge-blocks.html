

a: child
  
  b: child inside
  


c: child

e: child
  
e.f: child outside


g: child
g: default
  
g.h: child outside


i: child
i: default
  
  j: default inside block
  

